app/assets/javascripts/proxy_status/dhcp.js

```diff
@@ -0,0 +1,4 @@
+function activateDhcpTables(state) {
+    showProxies();
+    activateDatatables();
```

Comment:
maybe we should include setTabs(), and if we do, maybe we need a general status callback to trigger all of the above (as its seems common to most)?